src/AutoRest.CSharp/Mgmt/Output/MgmtObjectType.cs

```diff
@@ -221,7 +221,26 @@ public override ObjectTypeProperty GetPropertyForSchemaProperty(Property propert
 
         protected override string CreateDescription()
         {
-            return BuilderHelpers.CreateDescription(ObjectSchema) + BuilderHelpers.CreateExtraDescriptionWithDiscriminator(this);
+            return ObjectSchema.CreateDescription() + CreateExtraDescriptionWithDiscriminator();
+        }
+
+        public static readonly List<string> DiscriminatorDescFixedPart = new List<string> { "Please note ",
+            " is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.",
+            "The available derived classes include " };
+
+        protected virtual string CreateExtraDescriptionWithDiscriminator()
```

Comment:
BuilderHelpers is in the Commom namespace which is shared by data plane and mgmt plane, therefore we'd better put this method in Mgmt namespace since it only affects mgmt plane.